src/main/webapp/WEB-INF/web.xml

```diff
@@ -64,16 +64,6 @@ http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
         <url-pattern>/appstats/*</url-pattern>
     </servlet-mapping>
 
-    <security-constraint>
-        <web-resource-collection>
-            <web-resource-name>appstats</web-resource-name>
-            <url-pattern>/appstats/*</url-pattern>
-        </web-resource-collection>
```

Comment:
I'm not about this but using OAuth requires us to remove `<security-constraint>` part in `web.xml`. 
https://cloud.google.com/appengine/docs/java/tools/remoteapi#configuring_remote_api_on_an_app_engine_client